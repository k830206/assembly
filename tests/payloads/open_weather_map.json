{
  "coord": {
    "lon": 144.96,
    "lat": -37.81
  },
  "weather": [
    {
      "id": 804,
      "main": "Clouds",
      "description": "overcast clouds",
      "icon": "04n"
    }
  ],
  "base": "stations",
  "main": {
    "temp": 287.8,
    "feels_like": 284.73,
    "temp_min": 285.37,
    "temp_max": 290.37,
    "pressure": 1018,
    "humidity": 82
  },
  "visibility": 10000,
  "wind": {
    "speed": 5.1,
    "deg": 190
  },
  "clouds": {
    "all": 90
  },
  "dt": 1582206363,
  "sys": {
    "type": 1,
    "id": 9548,
    "country": "AU",
    "sunrise": 1582228522,
    "sunset": 1582276366
  },
  "timezone": 39600,
  "id": 2158177,
  "name": "Melbourne",
  "cod": 200
}