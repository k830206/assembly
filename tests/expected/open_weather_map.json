{
  "wind_speed": 18.36,
  "temperature_degrees": 14.65
}